lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -38,6 +39,37 @@ fi
 echo_summary "Performing sanity checks..."
 
 
+### Check if repositories are prepared for the upgrade
+# We need to ensure that Cloud5 main repository and update repos are in place
+
+# Skip the repository checks if they are defined in provisioner.json:
+# there might be non-default (and/or network) path
+if [ -f /etc/crowbar/provisioner.json ]; then
+  PROVISIONER_JSON=/etc/crowbar/provisioner.json
+  for repo in SLE-Cloud SUSE-Cloud-5-Pool SUSE-Cloud-5-Updates; do
```

Comment:
Actually, moving to `post` hurts for the admin server, since we won't validate that it will have latest updates (including latest upgrade script...)